app/businessTypes.service.ts

```diff
@@ -1,198 +1,39 @@
 import { Injectable } from '@angular/core';
-import { Http, Response, Headers} from '@angular/http';
+import { Http, Headers} from '@angular/http';
 import { Observable } from 'rxjs/Rx';
 import { BusinessType } from './businessType';
 import { BusinessCategory } from './businessCategory';
 
 @Injectable()
 export class BusinessTypeService {
-    private baseUrl: string = 'https://data.cityofgainesville.org/resource/pp75-zh6w.json';
+    // private baseUrl: string = 'https://data.cityofgainesville.org/resource/pp75-zh6w.json';
+    private baseUrl: string = 'http://localhost:8081/businesstypes.json';
```

Comment:
Shouldn't this be the open data portal URL and not a local server? How can we set this up?